benchmarking/es/insert_simple.js

```diff
@@ -3,8 +3,6 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 // Copyright (c) 2014 Mozilla Corporation
 //
```

Comment:
Extra comment line here that should be removed to be consistent with the rest of the commit